x-pack/platform/plugins/shared/lens/public/visualizations/heatmap/toolbar_component/style_settings.test.tsx

```diff
@@ -83,4 +83,135 @@ describe('heatmap style settings', () => {
     const updatedOrientationGroup = screen.queryByRole('group', { name: 'Orientation' });
     expect(updatedOrientationGroup).not.toBeInTheDocument();
   });
+
+  it('should update xSortPredicate when X-axis sort order is changed to descending (string column)', async () => {
```

Comment:
would be great to add a functional test too that changes the various predicates and checks the charts data sorting order to avoid any regression